### DISCUSSION TITLE
Supress Prints wiki.exists

### DISCUSSION THREAD

**Discussion by DanHarJor** [fastily/jwiki#40] [Q&A]
Hello, I want to use wiki.exists on thousands of strings.

Is there any way I can suppress the print out:
Apr 20, 2022 06:14:13 PM
INFO: [<Anonymous> @ en.wikipedia.org]: Checking to see if title exists: github

I think the printing will slow it down considerably. I would rather not wait. Is there any way I can supress this print out. Like with verbose = false or something.

Thank you

**Comment by benjaminkomen** [comment]
You could make your own pull request to change the logging level

**Reply by DanHarJor** [reply]
Thanks for your reply. I probably would do that but I only have a few days till my deadline so time is of the essence.
I managed to get it not to print by looking at your source code and copying what is returned by the method. 

It is still too slow for my application. 

Is wikipedia limiting the speed? It seems slower than my network connection. So I wonder if wikipedia only allows so many requests per second. 

I seen I can do a batch request. Then I noticed "private static final int groupQueryMax = 50;". Does this mean I can only do a batch request of 50 strings to see if they have pages? Is there a good reason for this or can I try and do larger batches?

Does it pull the entire page when I do one wiki.exists("string") call? Or does it only get the requested information?

I would very much appreciate any help you can offer.
thanks
